spec/models/user_spec.rb

```diff
@@ -7,6 +7,9 @@
   it {should validate_presence_of(:github_access_token)}
   it {should validate_uniqueness_of(:github_handle)}
   it {should validate_uniqueness_of(:github_access_token)}
+  # it {should validate_presence_of(:slack_handle)}
```

Comment:
do we need to leave these commented out lines in here?

Reply:
might put it back in later